.pre-commit-config.yaml

```
@@ -46,14 +48,7 @@ repos:
     rev: v2.7.1
     hooks:
       - id: prettier
-        exclude: |
-          (?x)(
-              ^lightning_api_access/ui/build|
-              README.md|
-              ^.github|
-              ^tests|
-              ^docs
-          )
+        exclude: lightning_api_access/ui/build
```

the whole UI needs to be excluded. 
```suggestion
        exclude: lightning_api_access/ui
```